tests/libnet/cluster.go

```diff
@@ -0,0 +1,38 @@
+package libnet
+
+import (
+	"fmt"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	netutils "k8s.io/utils/net"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func IsClusterDualStack(virtClient kubecli.KubevirtClient) (bool, error) {
```

Comment:
you might then want to replace `clusterSupportsIpv6` with this function ?

Reply:
Let's make a conscious effort to be aligned with the PRs effort please.

The scope / goal of this PR is to add a function that checks if a cluster is dual stack, then, unless the user explicitly says so, run a test checking that the cluster is indeed dual stack.

No, I don't want to replace any single stack ipv6 function with the code provided here, since I want to preserve the scope of this PR.